### DISCUSSION TITLE
Price on request

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#482] [_Discarded pitches]
### Current state

Price on request function can currently not be used.

The actual property prices should not be displayed by default if "Price on request" is set to "Yes". At the moment, this is only possible via template adjustments.

Currently we have to add the field “preisAufAnfrage” into the $dontEcho and then later into an if-loop so the value “Preis auf Anfrage” will be displayed instead the actual price.

```
$dontEcho = array("objekttitel", "objektbeschreibung", "lage", "ausstatt_beschr", "vermarktungsart", "objektart", "nutzungsart", "regionaler_zusatz", "ort", "plz", "land", "vermarktungsstatus", "preisAufAnfrage");

if ($field == "kaufpreis" || $field == "warmmiete") {
	if ($currentEstate['preisAufAnfrage'] == 'Ja') {
		$value = esc_html('Preis auf Anfrage', 'oo_theme');
	}
}
```

### Desired state
If the “Price on request” Checkbox is ticked in onOffice enterprise the value should display “Preis auf Anfrage” in the frontend of the website instead of the actual price.

**Comment by fredericalpers** [comment]
@dai-eastgate Please investigate and let me know how long this approx. will take.

**Reply by dai-eastgate** [reply]
@fredericalpers I need 3 days to fix and test it.
